runtime/doc/autocmd.txt

```diff
@@ -787,6 +787,32 @@ ModeChanged			After changing the mode. The pattern is
 		:au ModeChanged [vV\x16]*:* let &l:rnu = mode() =~# '^[vV\x16]'
 		:au ModeChanged *:[vV\x16]* let &l:rnu = mode() =~# '^[vV\x16]'
 		:au WinEnter,WinLeave * let &l:rnu = mode() =~# '^[vV\x16]'
+Progress						*Progress*
+				After a progress-msg is message is created via
+				`nvim_echo`. The pattern is matched against
+				title of the message. The data item in autocmd
+				callback contains following
+					msg_id: id of the message
+					message: text of the message
```

Comment:
use either "msg" or "message", not both ways. but maybe `text` could work.
```suggestion
					msg_id: id of the message
					text: message text
```

Reply:
Should we use `content` instead? ui-msg-show event uses `content` parameter for this

Reply:
ui event param names because they are not permanent (they are ordered args, not named). they are not a great source of best practices.